### PR TITLE
Rework example to avoid graphql warnings

### PR DESCRIPTION
**Pull request by shadowspawn**
Update example for command-arguments:
- remove the stale file showing deprecated extended `.description()`
- rework example as graphql complains about logging variable named "password"!

### CODE REVIEW

File: examples/argument.js [hunk 1]
```diff
@@ -1,24 +1,24 @@
...
-  .description('example program for argument')
-  .action((username, password) => {
-    console.log('username:', username);
-    console.log('password:', password);
+  .name('connect')
+  .argument('<server>', 'connect to the specified server')
+  .argument('[user]', 'user account for connection', 'guest')
+  .description('example program with argument descriptions')
+  .action((server, user) => {
+    console.log('username:', server);
```
Comment (abetomo) [inline examples/argument.js:15]: I think `'username'` would be better as `'server'` or something like that.

Comment (shadowspawn) [reply examples/argument.js:15]: Oops, missed that one! Thanks.

Comment (shadowspawn) [reply examples/argument.js]: Fixed

**Review by abetomo — APPROVED** [review]
👍

**Pull request merged**